Plugins/KnitBuildPlugin/KnitBuildPlugin.swift

```diff
@@ -102,7 +102,7 @@ extension XcodePluginContext {
 
         return .buildCommand(
             displayName: "Knit Plugin: Generate Knit files based on config \(configFilePath.description)",
-            executable: try self.tool(named: "KnitCommand").path,
+            executable: try self.tool(named: "knit-cli").path,
```

Comment:
Just out of curiosity did you try what happens if this line is the only change? I wonder if maybe the tool lookup here is supposed to be for the product (rather than the target). After the other name changes i suppose it works cause the product vs target is ambiguous.

Reply:
The build fails with the error `Plugin does not have access to a tool named ‘knit-cli’`

Reply:
Ok thanks for checking